keanu-python/tests/test_cast.py

```diff
@@ -0,0 +1,44 @@
+from keanu.cast import cast_to_double, cast_to_integer, cast_to_bool
+from keanu.vartypes import tensor_arg_types
+import pytest
+import numpy as np
+import pandas as pd
+
```

Comment:
Are there any tests you can do of exceptional cases? What happens when you try to cast `None` or a string, for example?